StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1651DoNotUsePlaceholderElements.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace StyleCop.Analyzers.DocumentationRules
+{
+    using System;
+    using System.Collections.Immutable;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using Microsoft.CodeAnalysis.Diagnostics;
+
+    /// <summary>
+    /// The element documentation for the element contains one or more &lt;placeholder&gt; elements.
+    /// </summary>
+    /// <remarks>
+    /// <para>A violation of this rule occurs when the element documentation contains &lt;placeholder&gt;
+    /// elements:</para>
+    ///
+    /// <code language="csharp">
+    /// /// &lt;summary&gt;
+    /// /// This method &lt;placeholder&gt;performs some operation&lt;/placeholder&gt;.
+    /// /// &lt;/summary&gt;
+    /// public void SomeOperation()
+    /// {
+    ///     ...
+    /// }
+    /// </code>
+    ///
+    /// <para>Placeholder elements should be reviewed and removed from documentation.</para>
+    /// </remarks>
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class SA1651DoNotUsePlaceholderElements : DiagnosticAnalyzer
+    {
+        /// <summary>
+        /// The ID for diagnostics produced by the <see cref="SA1651DoNotUsePlaceholderElements"/>
+        /// analyzer.
+        /// </summary>
+        public const string DiagnosticId = "SA1651";
+        private const string Title = "Do not use placeholder elements";
+        private const string MessageFormat = "Do not use placeholder elements";
+        private const string Category = "StyleCop.CSharp.DocumentationRules";
+        private const string Description = "The element documentation contains a <placeholder> element.";
+        private const string HelpLink = null;
```

Comment:
We'll get AppVeyor configured to build a SHFB site and migrate our links to point there.